src/pyOpenMS/pyopenms/_dataframes.py

```diff
@@ -339,6 +351,7 @@ def get_massql_df(self):
         'precmz': mass to charge of the precursor ion
         'ms1scan': number of the corresponding MS1 spectrum
         'charge': charge of the precursor ion
+        'ion': the ionic mobility of a peak if ion parameter is True
```

Comment:
It looks like the way it is currently implemented that it would show up in both MS1 and MS2? If that's true, move it up?